src/Util/ClassSourceManipulator.php

```diff
@@ -1002,19 +1002,20 @@ private function getEntityTypeHint($doctrineType)
             case 'string':
             case 'text':
             case 'guid':
+            case 'bigint':
```

Comment:
I think `decimal` should be moved here too

Reply:
True ! Fixed